src/handles/handle.rs

```diff
@@ -59,14 +59,23 @@ macro_rules! add_constant_handle {
 /// Object handle module
 ///
 pub mod object {
-    use crate::tss2_esys::{ESYS_TR_NONE, ESYS_TR_PASSWORD};
+    use crate::tss2_esys::{
+        ESYS_TR_NONE, ESYS_TR_PASSWORD, ESYS_TR_RH_ENDORSEMENT, ESYS_TR_RH_LOCKOUT,
+        ESYS_TR_RH_NULL, ESYS_TR_RH_OWNER, ESYS_TR_RH_PLATFORM, ESYS_TR_RH_PLATFORM_NV,
+    };
     impl_basic_handle!(
         /// General handle type.
```

Comment:
Not necessarily for this PR, but maybe this should mention that it's a wrapper for `ESYS_TR` handles.

Reply:
Yes, it should be mentioned. It is on my TODO list when it comes to improving the documentation.